lambda/__tests__/02.users-and-teams.test.ts

```diff
@@ -183,19 +183,19 @@ describe('users and teams', () => {
     it('should not allow non-admins to add users to their team', async () => {
       createMockAuth(TEAM_MEMBER_IDIR_USERID_01, TEAM_MEMBER_IDIR_EMAIL_01);
       const result = await addMembersToTeam(teamId, [{ idirEmail: 'test_user', role: 'member' }]);
-      expect(result.status).toEqual(401);
+      expect(result.status).toEqual(422);
```

Comment:
I think for the status codes, if you want to keep it as 401 for unauthorized you can do `throw {status: 401}` from anywhere in the code and the global error handler will use the status provided. You can add a message that way too, with `throw {status: 401, message: 'unauthorized'}`

Reply:
Thanks Jon that works and I thought of building some custom function or class to handle that but I remembered we do have a ticket to fix all the error handling so I just left it for now to keep it consistent with what we have